django_rq/queues.py

```diff
@@ -118,10 +141,11 @@ def get_queue(name='default', default_timeout=None, async=None,
 
     if default_timeout is None:
         default_timeout = QUEUES[name].get('DEFAULT_TIMEOUT')
-
-    return DjangoRQ(name, default_timeout=default_timeout,
-                    connection=get_connection(name), async=async,
-                    autocommit=autocommit)
+    if queue_class is None or queue_class == DjangoRQ:
```

Comment:
I think this should simply be (if you specify anything at all, no need to do anything special with it).
```python
if queue_class is None:
    queue_class = get_queue_class(QUEUES[name])
```